classes/class.tapestry-user-progress.php

```diff
@@ -89,6 +89,12 @@ public function complete()
         $this->_complete();
     }
 
+    public function completeQuestion($questionId)
+    {
+        $this->_checkUserAndPostId();
+        $this->completeQuestion($questionId);
```

Comment:
Shouldn't it be `_completeQuestion`? Seems like a deadly infinite loop.

Reply:
Thanks, fixed.